programs/openbook-v2/src/accounts_ix/create_open_orders_account.rs

```diff
@@ -10,13 +10,16 @@ pub struct CreateOpenOrdersAccount<'info> {
     pub delegate_account: Option<UncheckedAccount<'info>>,
     #[account(
         mut,
-        has_one = owner,
-        has_one = market
+        seeds = [b"OpenOrdersIndexer".as_ref(), owner.key().as_ref()],
+        bump = open_orders_indexer.bump,
+        realloc = OpenOrdersIndexer::space(open_orders_indexer.addresses.len()+1),
+        realloc::payer = payer,
+        realloc::zero = false,
```

Comment:
is`realloc::zero = false` safe? what happens if an user grows the account, then reduces it and finally grows it again?

Reply:
The test is doind that right?

Reply:
https://github.com/coral-xyz/anchor/pull/1943#issuecomment-1145985943